build/charts/theia/provisioning/datasources/create_table.sh

```diff
@@ -285,6 +285,24 @@ clickhouse client -n -h 127.0.0.1 <<-EOSQL
     ) engine=ReplicatedMergeTree('/clickhouse/tables/{shard}/{database}/{table}', '{replica}')
     ORDER BY (timeCreated);
 
+    --Create a table to store the Throughput Anomaly Detector results
+    CREATE TABLE IF NOT EXISTS tadetector_local (
+        sourceIP String,
+        sourceTransportPort UInt16,
+        destinationIP String,
+        destinationTransportPort UInt16,
+        protocolIdentifier UInt16,
+        flowStartSeconds DateTime,
+        flowEndSeconds String,
+        throughputStandardDeviation String,
+        algoType String,
+        algoCalc String,
+        Throughputs String,
```

Comment:
`Throughputs` -> `throughputs` or `throughput` if you only store a single throughput value in this column

Reply:
Also the datatype should not be `String` if you are storing throughput value

Reply:
ack